### PR TITLE
9 task 14 improving coverage refactor

### PR DESCRIPTION
**Pull request by tracyye029**
refactor state_spec.rb and county_spec.rb

### CODE REVIEW

File: spec/models/news_item_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe NewsItem, type: :model do
+  describe 'methods' do
+    describe '.find_for' do
+      it 'returns a representative_id news item' do
+        r = Representative.create(name: 'Brevin', ocdid: '01', title: 'Mayor')
+        ni = described_class.create(title: 'News Item', link: 'https://youtube.com', representative: r)
```
Comment (coccooma) [inline spec/models/news_item_spec.rb:10]: could use more detailed words as variable names

File: spec/models/news_item_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        expect(r.news_items).to include(ni)
+        expect(ni.representative).to eq(r)
+      end
+
+      it 'returns nil if representative_id has no news item' do
+        Representative.create(name: 'Brevin', ocdid: '01', title: 'Mayor')
+        found_ni = described_class.find_for(0o1)
```
Comment (coccooma) [inline spec/models/news_item_spec.rb:17]: same thing here

File: spec/models/news_item_spec.rb (hunk 1 above)
Comment (tracyye029) [reply spec/models/news_item_spec.rb:10]: No rubocop requires me to change detailed name to "described_class"

File: spec/models/news_item_spec.rb (hunk 2 above)
Comment (tracyye029) [reply spec/models/news_item_spec.rb:17]: rubocop says change detailed name to "described_class"

**Pull request merged**